src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx

```diff
@@ -39,7 +40,9 @@ export const IncidentsDetail = ({
       </header>
       <Wrapper>
         <FormTitle>Omschrijving</FormTitle>
-        <Paragraph strong>{text}</Paragraph>
+        <Paragraph strong data-testid="omschrijving-melding-mijn-meldingen">
```

Comment:
why is this test id added? normally not needed And don't see a test using it either

Reply:
removed, was there to make sure I was changing the right component.